src/popup/components/ProtocolIcon.vue

```diff
@@ -45,6 +47,9 @@ export default defineComponent({
         case PROTOCOL_BITCOIN: {
           return BitcoinIcon;
         }
+        case PROTOCOL_ETHEREUM: {
```

Comment:
I suggest to update the way how the list of icons are built to enforce adding the icon if new protocol is added. For example:
```
const iconsMap: Record<Protocol, Component> = {
   [PROTOCOL_AETERNITY]: AeternityIcon,
   ...
}

const selectedIcon = computed((): Component | undefined => iconsMap[props.protocol])
```

This way the TS engine will inform us about adding the icon when new protocol is added.